content/articles/statistical-hypothesis-testing-python-implementation/index.md

```diff
@@ -0,0 +1,506 @@
+
+***The smaller our P-values is the more delicate it is to accept our null hypothesis.***
+
+To calculate our P-value, there should be a certain level of significance to our ground truth or initial assumption, so a smaller P-value is telling us there are more reasons why we should accept our alternative hypothesis.
+
+P-value is within the range of 0 and 1, but the threshold value is always set at 0.05, the threshold value can also be called *alpha*, alpha is always set before the experiments in order to avoid bias. 
+Alpha is the level we set our p-value at inorder to ascertain whether to accept our null hypothesis or not.
+Our alpha value is the probability of making a wrong decision when the ground truth(H0) is true. It is always calculated as 
+```
+alpha = 1 - confidence level
+```
+if the confidence level from what we've seen in our data is 95%, then 
+```
+alpha = 1 - 0.95
+```
+which is 0.05.
+
+Why is 95% confidence level the most accustomed?
+
+Since we know that alpha level is the probability of encountering a Type I error(which is rejecting H0 despite being true), would it not make sense to make it as bitsy as possible. Imagine our alpha value being set to 10%, obviously, there is an increasing chance that we will make the mistake of rejecting the null hypothesis, but it must not be too small as well, if we set the alpha value as 1%, it will be too small that our decision will be reeking of bias. Hence, the standard has been 5%.
+
+While inplementing statistical hypothesis testing, there are some methodologies that can be employed, in this tutorial, I will be touching on three methods of carrying out statistical hypothesis testing.
+
+### 5. [Statistical Techniques for Hypothesis Testing](#5-statistical-techniques-for-hypothesis-testing)
+The three methods to be discussed are:
+1. Chi-Square Test
+2. T-Test
+3. ANOVA Test
+
+### 6. [Chi-square test - Python implementation.](#6-chi-square-test-python-implementation)
+
+Chi-square test is used to perform testing on two categorical variables in our data population. It is focused on looking for an important relationship between two categorical variables.
+For example, comparing the aftereffect of malaria drug A, malaria drug B, and malaria drug C.
+Chi-square is a tool for checking how divergent the observed frequency is from the expected frequency.
+With Chi-Square, we accept the null hypothesis when variable A and variable B are not dependent, i.e. there is no relationship between them.
+We reject the null hypothesis(or accept the alternate hypothesis) when variable A depends on variable B, i.e. there is a relationship between them. The formula for calculating chi-square test is: 
+
+`Chi-square = Σ[(observed value - expected value)² / expected value]`
+
```

Comment:
Like I've said, it's better if formulae are taken from Google Images or typing it in "Equation Mode" in Microsoft Word.

Reply:
Is anything wrong with this format?